src/gl-matrix/quat.js

```diff
@@ -193,6 +193,35 @@ quat.setAxisAngle = function(out, axis, rad) {
 };
 
 /**
+ * Gets the rotation axis and angle for a given
+ *  quaternion. If a quaternion is created with
+ *  setAxisAngle, this method will return the same
+ *  values as providied in the original parameter list
+ *  OR functionally equivalent values.
+ * Example: The quaternion formed by axis [0, 0, 1] and
+ *  angle -90 is the same as the quaternion formed by
+ *  [0, 0, 1] and 270. This method favors the latter.
+ * @param  {quat} q     Quaternion to be decomposed
+ * @param  {vec3} axis  Vector receiving the axis of rotation
+ * @return {Number}     Angle, in radians, of the rotation
+ */
+quat.getAxisAngle = function(q, axis) {
```

Comment:
I think that for the sake of consistency this should be: `function(out_axis, q)`. I'll admit it reads a little funny, but every other function in the library has the output vector/mat/quat as the first argument so we probably shouldn't break that pattern here.